backend/pcloud/pcloud.go

```diff
@@ -1033,6 +1036,224 @@ func (f *Fs) Shutdown(ctx context.Context) error {
 	return nil
 }
 
+// changeNotify implements fs.Features.ChangeNotify
+func (f *Fs) changeNotify(ctx context.Context, notify func(string, fs.EntryType), ch <-chan time.Duration) {
```

Comment:
This should be called `ChangeNotify`

and here

https://github.com/rclone/rclone/blob/976aa6b41680211e06b2821e67ef92de10c376e2/backend/pcloud/pcloud.go#L1403

Add an interface notification for the ChangeNotify method